tgui/packages/common/collections.js

```diff
@@ -0,0 +1,271 @@
+/**
+ * @file
+ * @copyright 2020 Aleksej Komarov
+ * @license MIT
+ */
+
+/**
+ * Converts a given collection to an array.
```

Comment:
You can give it an object, and it will return an array consisting only of values, discarding all the keys. Collection is pretty much anything iterable.